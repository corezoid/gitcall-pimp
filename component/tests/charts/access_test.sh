#!/usr/bin/env bash
$CZ helm template | sed -n '/access\.yaml/,/Source\:/p'
