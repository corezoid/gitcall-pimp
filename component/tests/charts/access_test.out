# Source: gitcall_pimp_release/charts/gitcall_pimp/templates/access.yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: "a_cluster_name"
  labels:
    release: "RELEASE-NAME"
    app: "gitcall"
    tier: "pimp"
rules:
- apiGroups: [""]
  resources: ["namespaces"]
  verbs: ["get", "list", "create", "delete", "update"]
- apiGroups: [""]
  resources: ["pods"]
  verbs: ["get", "list", "create", "delete", "update"]
- apiGroups: ["extensions", "apps"]
  resources: ["deployments"]
  verbs: ["get", "list", "create", "delete", "update"]
- apiGroups: [""]
  resources: ["services"]
  verbs: ["get", "create", "delete", "update"]
- apiGroups: ["autoscaling"]
  resources: ["horizontalpodautoscalers"]
  verbs: ["get", "create", "delete", "update"]
- apiGroups: [""]
  resources: ["configmaps"]
  verbs: ["get", "list", "create", "delete", "update", "patch"]
- apiGroups: [""]
  resources: ["secrets"]
  verbs: ["get", "list", "create", "delete", "update"]
---
# Source: gitcall_pimp_release/charts/gitcall_pimp/templates/access.yaml
